apps/addons/tests/test_models.py

```diff
@@ -296,6 +297,26 @@ def test_new_featured(self):
         f = Addon.objects.featured(amo.THUNDERBIRD)
         assert not f.exists()
 
+    def test_filter_for_many_to_many(self):
+        addon = Addon.objects.get(pk=3615)
+        collection = addon.collections.first()
```

Comment:
Can you:
- get the pk
- Use `Collection.objects.get(pk=collection_pk)` like you do below
- Add `.nocache()` to that query
- Repeat with `Version` for the test below

That way we are consistent and we know cache-machine is not interfering.